PantryPal/package-lock.json

```diff
@@ -1,8 +1,9706 @@
 {
   "name": "pantrypal",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Should we update the git ignore to remove this file too? or is this meant to be included

Reply:
NVM, looking into it, we just commit it every time, but ignore it
[source](https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5)